db/migrate/20250106111536_add_title_to_content_block_editions.rb

```diff
@@ -10,4 +10,4 @@ def down
       t.remove :title
     end
   end
-end
\ No newline at end of file
+end
```

Comment:
I think this can be squashed into the original commit?

Reply:
oh yep, good spot!